docs/reference/benchmarks.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: Benchmarks
+nav_order: 4
+permalink: /reference/benchmarks/
+parent: Reference
+---
+
+This page describes each benchmark that is part of fuzzbench.
+
+The table below was generated using benchmarks.py.
+
+| Benchmark                   | Fuzz target            | Has dictionary? | Number of seed inputs | Number of progam edges | Size (MB) |
```

Comment:
Should we add a Dictionary size too?

Reply:
Maybe if someone asks for it but I'd rather not get in the business of dictionary parsing (simple as they are) here.